src/EFCore/DbContextOptions.cs

```diff
@@ -86,7 +86,7 @@ public virtual TExtension GetExtension<TExtension>()
         public virtual void Freeze() => IsFrozen = true;
 
         /// <summary>
-        ///     Returns true if <see cref="Freeze" />. has been called. A frozen options object cannot be further
+        ///     Returns <see langword="true"/> if <see cref="Freeze" />. has been called. A frozen options object cannot be further
```

Comment:
```suggestion
        ///     Returns <see langword="true"/> if <see cref="Freeze" /> has been called. A frozen options object cannot be further
```

Plus add `.` to end of next line.